rules-tests/CodeQuality/Rector/FuncCall/AssertFuncCallToPHPUnitAssertRector/Fixture/skip_in_static_method.php.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\CodeQuality\Rector\FuncCall\AssertFuncCallToPHPUnitAssertRector\Fixture;
+
+use PHPUnit\Framework\TestCase;
+
+final class SkipInStaticMethodTest extends TestCase
+{
+    private static function getExceptionMessage(string $fqcn): string
+    {
+        $exception = new $fqcn();
+        assert($exception instanceof \Exception);
```

Comment:
These should be handled too. The static call on `Assert::..` should be used instead.

Reply:
I see :+1: